src/Components/DiaryCard.js

```diff
@@ -26,36 +26,23 @@ function DiaryCard({title, subtitle, description, color }) {
     //Change String Size
     const resultString = more ? description.slice(0, 100) + '...' : description;
 
-    function ShowMore() {
-        setMore(false);
+    function showMore() {
+        setMore(!more)
     }
 
-    function ShowLess() {
-        setMore(true)
-    }
 
     return (
         <div>
-            <Card className={classes.root} style={{backgroundColor:`${color}`}}>
+            <Card className={classes.root} style={{ backgroundColor: `${color}` }}>
                 <CardContent>
                     <Typography variant="h5">{title}</Typography>
                     <Typography color="textSecondary">{subtitle}</Typography>
                     <Typography variant="body2" className={classes.description}>{resultString}</Typography>
                 </CardContent>
 
-
-                {
-                    more ?
-
-                        <CardActions>
-                            <Button size="small" onClick={ShowMore}>Show More</Button>
-                        </CardActions>
-                        :
-
-                        <CardActions>
-                            <Button size="small" onClick={ShowLess}>Show Less</Button>
-                        </CardActions>
-                }
+                <CardActions>
+                    <Button size="small" onClick={ShowMore}>Show {more ? "MORE" : "LESS"}</Button>
```

Comment:
Is this working? Your function is `showMore` but you have called it as `ShowMore` here. I think you should recieve an error `ShowMore is not defined`